gocourse02/main.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	zookyper := Zookyper{
+		Name:   "Mukola",
+		IsBusy: false,
+	}
+	zookyper.PrintName()
+
+	cage := Cage{
+		Number:   1,
+		IsBroken: true,
+	}
+
+	fmt.Printf("Cage number %d state =>%t\n", cage.Number, cage.IsBroken)
+	if !zookyper.IsBusy && cage.IsBroken {
+		cage.FixCage()
+	}
+	fmt.Printf("Cage number %d state =>%t\n", cage.Number, cage.IsBroken)
+
+	var sirko = Animal{
+		Name:   "Sirko",
+		Number: 1,
+	}
+	fmt.Printf("Animal with name %s is istting in cage number %d\n", sirko.Name, sirko.Number)
+
+	var sirko_son = sirko.Duplicate("Sirko junior", 1)
+	fmt.Printf("Animal with name %s is istting in cage number %d\n", sirko_son.Name, sirko_son.Number)
+
+}
+
+type Zookyper struct {
```

Comment:
Мені здається, "наглядач" не так пишеться. Ось https://www.merriam-webster.com/dictionary/zookeeper

```suggestion
type Zookeeper struct {
```

Виправ тут і всюди по коду, будь ласка.